CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # v0.4.0 (Unreleased)
 
 NEW FEATURES:
+
 * Add support for importing from [glock](https://github.com/robfig/glock) based projects. ([#1422](https://github.com/golang/dep/pull/1422)
 * Add support for importing from [govendor](https://github.com/kardianos/govendor) based projects. ([#815](https://github.com/golang/dep/pull/815)
 * Allow override of cache directory location using environment variable `DEPCACHEDIR`. ([#1234](https://github.com/golang/dep/pull/1234))
 * Add support for template output in `dep status`. ([#1389](https://github.com/golang/dep/pull/1389)
 * Each element in a multi-item TOML array is output on its own line. ([#1461](https://github.com/golang/dep/pull/1461)
+* Added `install.sh` script.
```

Comment:
Please follow the pattern of the other CHANGELOG entries - include a link to this PR.

Reply:
my bad! done 👍